Sources/AppcuesKit/Presentation/Debugger/Panel/DebugViewModel.swift

```diff
@@ -86,18 +87,58 @@ internal class DebugViewModel: ObservableObject {
         return currentUserID
     }
 
-    init(networking: Networking, accountID: String, applicationID: String, currentUserID: String, isAnonymous: Bool) {
+    init(
+        networking: Networking,
+        accountID: String,
+        applicationID: String,
+        currentUserID: String,
+        isAnonymous: Bool,
+        bundleFontsPath: String?
+    ) {
         self.networking = networking
         self.accountID = accountID
         self.applicationID = applicationID
         self.currentUserID = currentUserID
         self.isAnonymous = isAnonymous
         self.userIdentified = !currentUserID.isEmpty
+        self.bundleFontsPath = bundleFontsPath
 
         connectedStatus.action = Action(symbolName: "arrow.triangle.2.circlepath") { [weak self] in self?.ping() }
         deepLinkStatus.action = Action(symbolName: "arrow.triangle.2.circlepath") { [weak self] in self?.verifyDeepLink() }
     }
 
+    func fonts() -> [(title: String, names: [String])] {
+        let uiAppFontPaths = (Bundle.main.infoDictionary?["UIAppFonts"] as? [String] ?? []).compactMap { Bundle.main.url(forResource: $0, withExtension: nil) }
+        let customBundleFontPaths = Bundle.main.urls(forResourcesWithExtension: "ttf", subdirectory: bundleFontsPath) ?? []
```

Comment:
This is the important link to read from the bundle fonts. The rest of this is just refactored from `DebugFontUI`.